backend/de.metas.elasticsearch/src/main/java/de/metas/fulltextsearch/query/FTSSearchService.java

```diff
@@ -170,6 +174,35 @@ private Evaluatee createQueryEvalCtx(
 			builder.put(ESQueryTemplate.PARAM_orgFilter, buildOrgIdsFilterPart(request.getUserRolePermissionsKey()));
 		}
 
+		if (request.hasDefaultFilterParams())
+		{
+			for(final Map.Entry<String, Object> entry : request.getDefaultFilterParams().entrySet() )
+			{
+				final CtxName fieldName = CtxNames.parseNotNull(entry.getKey());
+				if (jsonQueryTemplate.isParameterRequired(fieldName))
+				{
+					if(entry.getValue() instanceof Boolean)
+					{
+						builder.put(fieldName, buildBooleanFilterPart(entry.getKey(), (boolean) entry.getValue()));
+					}
+					else if(entry.getValue() instanceof String)
+					{
+						builder.put(fieldName, buildStringFilterPart(entry.getKey(), (String) entry.getValue()));
+					}
+					else if(entry.getValue() instanceof Integer)
+					{
+						builder.put(fieldName, buildIntegerFilterPart(entry.getKey(), (Integer) entry.getValue()));
+					}
+					else
+					{
+						// In FTS Config Query, but currently not used in DefaultFilter
+						builder.put(fieldName, "");
```

Comment:
hmmm, i would suggest
* handle the missing/null case separatelly
* if the value is not boolean/String/Integer then handle it as String (loud thinking) or throw an exception